src/base/static/components/organisms/main-map.tsx

```diff
@@ -166,6 +193,13 @@ interface State {
   popupLatitude: number | null;
   popupLongitude: number | null;
   mapViewport: MapViewport;
+  measurementTool: {
```

Comment:
Because map widgets now live in containers, it was necessary to break apart the measurement tool's controls and overlay. The overlay needs to render as a child of the map and not the widget container.

To support this, I moved most of the measurement widget's state to `MainMap`. I considered moving it to Redux, but it seemed unnecessary since nothing outside the map needs to know about it.

Hopefully this doesn't add more weight than we should have in `MainMap`, though...